include/avif/avif.h

```diff
@@ -85,8 +85,11 @@ extern "C" {
     ((AVIF_VERSION_MAJOR * 1000000) + (AVIF_VERSION_MINOR * 10000) + (AVIF_VERSION_PATCH * 100) + AVIF_VERSION_DEVEL)
 
 typedef int avifBool;
-#define AVIF_TRUE 1
-#define AVIF_FALSE 0
+enum avifTrueFalse
```

Comment:
Is it necessary to name the enum? Could it be the following instead?

```c
typedef enum avifBool {
    AVIF_FALSE = 0,
    AVIF_TRUE = 1
} avifBool;
```

Reply:
avifBool is used a lot all over and we have casts from bools to avifBool in tests all over so that creates new failures. We can do 
```
typedef int avifBool;
typedef enum avifTrueFalse {
    AVIF_FALSE = 0,
    AVIF_TRUE = 1
} avifTrueFalse;
```
I could not find any warning when using an int instead of an enum (which makes sense) but I could find a warning when casting an enum to a different enum: https://clang.llvm.org/docs/DiagnosticsReference.html#wenum-conversion